runtime-display/egl-wayland/spec

```diff
@@ -1,4 +1,4 @@
-VER=1.1.7
+VER=1.1.13
 SRCS="tbl::https://github.com/NVIDIA/egl-wayland/archive/$VER.tar.gz"
```

Comment:
Please try not to use automatically packaged tarballs, use Git tags instead.

Reply:
Fixed.